felles/oidc/src/main/java/no/nav/vedtak/sikkerhet/oidc/config/impl/OidcProviderConfig.java

```diff
@@ -83,17 +83,17 @@ private static Set<OpenIDConfiguration> hentConfig() {
         }
 
         // Azure - ikke alle apps trenger denne (tokenx-apps)
-        var azureKonfigUrl = getAzureProperty(AzureProperty.AZURE_APP_WELL_KNOWN_URL);
+        var azureKonfigUrl = ENV.getProperty(AzureProperty.AZURE_APP_WELL_KNOWN_URL.name());
```

Comment:
Har vi noen apps som ikke har azure:enabled:true i naiserator? 
Burde nesten vært en exception og stop() hvis man mangler både azure og tokenx

Reply:
En ting er å hente/veksle token og en annen ting er å validere. Det at man har utgående kall betyr ikke at man må ha inngående. Tror fp-oversikt er en godt eksempel hvor man får inngående TokenX fra selvbetjening men så kaller videre til backendene med AzureCC

Reply:
OG - hvis man ikke har tokenx og ikke har azuread - da har man en dud app som gjør lite :-) 
Fp-oversikt kaller med azureAd når den er asynkron fra brukerkontekst (prosesstask) (+ noen spesialendepunkt). Ellers er det pass in TokenX i adaptive

Reply:
fp-dokgen f.eks ;) men heldigvis bruker vi ikke fp-felles der heler. Ja jeg er enig bør være minst en av dem være satt. Men mulig vi bør konfigurere det med en annen param enn WELL_KNOWN_URL ?

Reply:
Bare å ta den inn. Samme hvilken - bare sjekke på en nais-injisert property